documentation/migration-guide.md

```diff
@@ -68,27 +68,55 @@ If you're using the `rdme` GitHub Action, update your GitHub Actions workflow fi
     rdme: openapi validate petstore.json
 ```
 
-#### Step 3: Address `v10` Breaking Changes
+#### Step 2: Address `v10` Breaking Changes
 
 1. **Enable Bi-Directional Syncing** _(recommended)_
 
    We recommend setting up [bi-directional syncing](https://docs.readme.com/main/docs/bi-directional-sync) for managing your Markdown files, API definitions and project hierarchy.
 
 2. **Command Replacements**
-   - Replace: `openapi` → `openapi upload` (see more in step 3 below)
+   - Replace: `changelogs` → `changelog upload` (see more in step 3 below)
```

Comment:
Can these "see more in step X" callouts be anchor links?

Reply:
done: https://github.com/readmeio/rdme/pull/1287/commits/e02a973cbf67eece82f3b31b217d3f2386c3b048